projects/saturn/src/main/java/io/fairspace/saturn/config/ViewsConfig.java

```diff
@@ -1,18 +1,30 @@
 package io.fairspace.saturn.config;
 
 import java.util.*;
+import java.util.function.Function;
+import java.util.stream.Collectors;
 
 import com.fasterxml.jackson.annotation.*;
 import com.fasterxml.jackson.databind.*;
 import com.fasterxml.jackson.dataformat.yaml.*;
 import jakarta.validation.constraints.*;
 
 public class ViewsConfig {
+
+    private Map<String, View> viewConfig;
```

Comment:
Could you rename this to something like `viewConfigMap` or `nameToViewConfigMap`? 
It is confusing that getViewConfig gives you `View`, buut viewConfig is a map of names to Views.

Reply:
agree

Reply:
renamed